### PR TITLE
xilinx: Adapt wait times with JTAG frequency.

### PR DESCRIPTION
**Pull request by UweBonnes**
Allowed frequency is up to 10 MHz according to xc95XXXxl_1532.bsd

### CODE REVIEW

File: src/xilinx.cpp [hunk 1]
```diff
@@ -461,11 +461,11 @@ bool Xilinx::flow_erase()
...
 	_jtag->shiftIR(XC95_ISC_ERASE, 8);
 	_jtag->shiftDR(xfer_buf, NULL, 18);
-	_jtag->toggleClk(2000000);
+	_jtag->toggleClk((_jtag->getClkFreq() * 400) / 1000);
 	_jtag->shiftDR(NULL, xfer_buf, 18);
 	if ((xfer_buf[0] & 0x03) != 0x01) {
-		printError("FAIL");
-		return false;
+	    printError("FAIL");
+	    return false;
```
Comment (trabucayre) [inline src/xilinx.cpp:468]: tab

**Comment by trabucayre** [comment]
LGTM. just two wrong indent :)

**Comment by UweBonnes** [comment]
Hopefully fixed

**Comment by trabucayre** [comment]
Applied to master.
Thanks

**Pull request merged**